docs/articles/documentation/reference/testcafe-api/browserconnection/README.md

```diff
@@ -11,11 +11,11 @@ A connection to a [remote browser](../../../guides/concepts/browsers.md#browsers
 
 Use this object to run tests on a remote or a mobile device.
 
-`BrowserConnection` is created with the [testCafe.createBrowserConnection](../testcafe/createbrowserconnection.md) function.
+You can create a new `BrowserConnection` with the [testCafe.createBrowserConnection](../testcafe/createbrowserconnection.md) function.
 
-The [browserConnection.url](url.md) property returns a URL the remote browser should visit in order to connect to the [TestCafe server instance](../testcafe/README.md).
+The [browserConnection.url](url.md) property returns a URL that remote browsers can visit to connect to the [TestCafe server instance](../testcafe/README.md).
```

Comment:
"browsers can visit to connect to" This does not make sense

Reply:
I meant:

> ... a **URL** that remote browsers can visit to connect to...

Maybe that's hard to parse.  What about.

> Remote browsers can visit a URL returned by the `browserConnection.url` property  to connect to a [TestCafe server instance](../testcafe/README.md).

OR

> The `browserConnection.url` property returns a URL. Remote browsers can visit this URL to connect to a TestCafe server instance.

Reply:
The browserConnection.url property returns a URL. Remote browsers can use this URL to connect to a TestCafe server instance.